diff --git a/node_modules/materia-widget-development-kit/express.js b/node_modules/materia-widget-development-kit/express.js
index 2caa799..049aaf1 100644
--- a/node_modules/materia-widget-development-kit/express.js
+++ b/node_modules/materia-widget-development-kit/express.js
@@ -18,7 +18,7 @@ const outputPath           = path.join(process.cwd(), 'build') + path.sep
 // Webpack middleware setup
 const webpack              = require('webpack');
 const webpackDevMiddleware = require('webpack-dev-middleware');
-const config               = require(path.resolve(process.cwd(), './webpack.config.js'));
+const config               = require(path.resolve(process.cwd(), './webpack.config.cjs'));
 const compiler = webpack(config);


